tests/test_pymediainfo.py

```diff
@@ -112,6 +117,33 @@ def test_parse_non_existent_path_pathlib(self):
         path = self.pathlib.Path(data_dir) / "this file does not exist"
         self.assertRaises(FileNotFoundError, MediaInfo.parse, path)
 
+
+class MediaInfoTestFilenameTypes(unittest.TestCase):
+    def test_convert_filename_str(self):
+        path = os.path.join(data_dir, "test.txt")
+        filename, url = convert_filename(path)
+        self.assertEqual(filename, path)
+
+    @pytest.mark.skipif(
+        sys.version_info < (3, 4),
+        reason="Were testing the pathlib route of execution, but pathlib only shipped with py3.4+")
```

Comment:
I'm gonna `importorskip` pathlib as I did for the other pathlib test, I might even merge those test classes. Pathlib can be available for older versions via backports.